src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1322,161 +1210,62 @@ public StringBuilder Insert(int index, char[] value, int startIndex, int charCou
-        // The capacity is adjusted as needed. 
-        // 
         [CLSCompliant(false)]
-        public StringBuilder Insert(int index, ulong value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, ulong value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to this string builder with value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the
-        // new text.  The capacity is adjusted as needed. If value equals String.Empty, the
-        // StringBuilder is not changed. No changes are made if value is null.
-        // 
-        public StringBuilder Insert(int index, Object value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            if (null == value)
-            {
-                return this;
-            }
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, Object value) => value == null ? this : Insert(index, value.ToString(), 1);
 
-        public StringBuilder AppendFormat(String format, Object arg0)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return AppendFormatHelper(null, format, new ParamsArray(arg0));
-        }
+        public StringBuilder AppendFormat(String format, Object arg0) => AppendFormatHelper(null, format, new ParamsArray(arg0));
 
-        public StringBuilder AppendFormat(String format, Object arg0, Object arg1)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return AppendFormatHelper(null, format, new ParamsArray(arg0, arg1));
-        }
+        public StringBuilder AppendFormat(String format, Object arg0, Object arg1) => AppendFormatHelper(null, format, new ParamsArray(arg0, arg1));
 
-        public StringBuilder AppendFormat(String format, Object arg0, Object arg1, Object arg2)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return AppendFormatHelper(null, format, new ParamsArray(arg0, arg1, arg2));
-        }
+        public StringBuilder AppendFormat(String format, Object arg0, Object arg1, Object arg2) => AppendFormatHelper(null, format, new ParamsArray(arg0, arg1, arg2));
 
         public StringBuilder AppendFormat(String format, params Object[] args)
         {
             if (args == null)
             {
                 // To preserve the original exception behavior, throw an exception about format if both
                 // args and format are null. The actual null check for format is in AppendFormatHelper.
-                throw new ArgumentNullException((format == null) ? nameof(format) : nameof(args));
+                string paramName = format == null ? nameof(format) : nameof(args);
```

Comment:
I think this was fine before, but in any case, parens around `format == null` please